recipes/blake2/all/makefile

```diff
@@ -0,0 +1,66 @@
+CFLAGS          += $(CONAN_CFLAGS)
```

Comment:
Where did you find it? Could you port to CMake instead?

Reply:
Are you asking about the makefile?

Reply:
If it only works on Linux, I see no problem running a makefile. Otherwise, I suggest you providing a cmake file. I can help you if you are new to cmake.